### ISSUE TITLE
10play / Tenplay .com.au issue

### ISSUE THREAD

**Issue by DarrenPIngram** [yt-dlp/yt-dlp#1692]
### Checklist

- [X] I'm reporting a broken site
- [X] I've verified that I'm running yt-dlp version **2021.11.10.1**. ([update instructions](https://github.com/yt-dlp/yt-dlp#update))
- [X] I've checked that all provided URLs are alive and playable in a browser
- [X] I've checked that all URLs and arguments with special characters are [properly quoted or escaped](https://github.com/ytdl-org/youtube-dl#video-url-contains-an-ampersand-and-im-getting-some-strange-output-1-2839-or-v-is-not-recognized-as-an-internal-or-external-command)
- [X] I've searched the [bugtracker](https://github.com/yt-dlp/yt-dlp/issues?q=) for similar issues including closed ones. DO NOT post duplicates
- [X] I've read the [guidelines for opening an issue](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#opening-an-issue)
- [X] I've read about [sharing account credentials](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#are-you-willing-to-share-account-details-if-needed) and I'm willing to share it if required

### Region

Australia

### Description

I believe that I have done everything correctly. I've also tried with .netrc and the cookie file was downloaded with a Chrome plugin, copied to the server and YT-dlp seems to have rewritten it to a format it is happy with. 

The URLs in url.txt work in a browser.

I am not sure what else to give at this point.

In netrc I've tried both machine tenplay and 10play (it errored out also doing a test with the word tenplay).

### Verbose log

```shell
yt-dlp -Uv --netrc --cookies /home/capture-au/scripts/cookies.txt --batch-file /home/capture-au/url.txt
[debug] Command-line config: ['-Uv', '--netrc', '--cookies', '/home/capture-au/scripts/cookies.txt', '--batch-file', '/home/capture-au/url.txt']
[debug] Batch file urls: ['https://10play.com.au/mirror-mirror/episodes/season-1/episode-2/tpv210909jrytv', 'https://10play.com.au/mirror-mirror/episodes/season-1/episode-1/tpv211105pbzld']
[debug] Encodings: locale UTF-8, fs utf-8, out utf-8, err utf-8, pref UTF-8
[debug] yt-dlp version 2021.11.10.1
[debug] Lazy loading extractors is disabled
[debug] Python version 3.9.8 (CPython 64bit) - Linux-5.10.0-9-amd64-x86_64-with-glibc2.31
[debug] exe versions: ffmpeg 4.4.1 (setts), ffprobe 4.4.1, rtmpdump 2.4
[debug] Optional libraries: Cryptodome, mutagen, sqlite, websockets
[debug] Proxy map: {}
yt-dlp is up to date (2021.11.10.1)
[debug] [TenPlay] Extracting URL: https://10play.com.au/mirror-mirror/episodes/season-1/episode-2/tpv210909jrytv
[TenPlay] tpv210909jrytv: Getting bearer token
ERROR: [TenPlay] tpv210909jrytv: tpv210909jrytv: Failed to parse JSON  (caused by JSONDecodeError('Expecting value: line 1 column 1 (char 0)')); please report this issue on  https://github.com/yt-dlp/yt-dlp . Make sure you are using the latest version; see  https://github.com/yt-dlp/yt-dlp  on how to update. Be sure to call yt-dlp with the --verbose flag and include its complete output. (caused by JSONDecodeError('Expecting value: line 1 column 1 (char 0)')); please report this issue on  https://github.com/yt-dlp/yt-dlp . Make sure you are using the latest version; see  https://github.com/yt-dlp/yt-dlp  on how to update. Be sure to call yt-dlp with the --verbose flag and include its complete output.
Traceback (most recent call last):
  File "/home/capture-au/.local/lib/python3.9/site-packages/yt_dlp/extractor/common.py", line 987, in _parse_json
    return json.loads(json_string)
  File "/home/linuxbrew/.linuxbrew/opt/python@3.9/lib/python3.9/json/__init__.py", line 346, in loads
    return _default_decoder.decode(s)
  File "/home/linuxbrew/.linuxbrew/opt/python@3.9/lib/python3.9/json/decoder.py", line 337, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/home/linuxbrew/.linuxbrew/opt/python@3.9/lib/python3.9/json/decoder.py", line 355, in raw_decode
    raise JSONDecodeError("Expecting value", s, err.value) from None
json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/capture-au/.local/lib/python3.9/site-packages/yt_dlp/extractor/common.py", line 596, in extract
    ie_result = self._real_extract(url)
  File "/home/capture-au/.local/lib/python3.9/site-packages/yt_dlp/extractor/tenplay.py", line 65, in _real_extract
    _token = self._get_bearer_token(content_id)
  File "/home/capture-au/.local/lib/python3.9/site-packages/yt_dlp/extractor/tenplay.py", line 55, in _get_bearer_token
    data = self._download_json('https://10play.com.au/api/user/auth', video_id, 'Getting bearer token', headers={
  File "/home/capture-au/.local/lib/python3.9/site-packages/yt_dlp/extractor/common.py", line 976, in _download_json
    res = self._download_json_handle(
  File "/home/capture-au/.local/lib/python3.9/site-packages/yt_dlp/extractor/common.py", line 962, in _download_json_handle
    return self._parse_json(
  File "/home/capture-au/.local/lib/python3.9/site-packages/yt_dlp/extractor/common.py", line 991, in _parse_json
    raise ExtractorError(errmsg, cause=ve)
yt_dlp.utils.ExtractorError: tpv210909jrytv: Failed to parse JSON  (caused by JSONDecodeError('Expecting value: line 1 column 1 (char 0)')); please report this issue on  https://github.com/yt-dlp/yt-dlp . Make sure you are using the latest version; see  https://github.com/yt-dlp/yt-dlp  on how to update. Be sure to call yt-dlp with the --verbose flag and include its complete output.
Traceback (most recent call last):
  File "/home/capture-au/.local/lib/python3.9/site-packages/yt_dlp/extractor/common.py", line 987, in _parse_json
    return json.loads(json_string)
  File "/home/linuxbrew/.linuxbrew/opt/python@3.9/lib/python3.9/json/__init__.py", line 346, in loads
    return _default_decoder.decode(s)
  File "/home/linuxbrew/.linuxbrew/opt/python@3.9/lib/python3.9/json/decoder.py", line 337, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/home/linuxbrew/.linuxbrew/opt/python@3.9/lib/python3.9/json/decoder.py", line 355, in raw_decode
    raise JSONDecodeError("Expecting value", s, err.value) from None
json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/capture-au/.local/lib/python3.9/site-packages/yt_dlp/extractor/common.py", line 596, in extract
    ie_result = self._real_extract(url)
  File "/home/capture-au/.local/lib/python3.9/site-packages/yt_dlp/extractor/tenplay.py", line 65, in _real_extract
    _token = self._get_bearer_token(content_id)
  File "/home/capture-au/.local/lib/python3.9/site-packages/yt_dlp/extractor/tenplay.py", line 55, in _get_bearer_token
    data = self._download_json('https://10play.com.au/api/user/auth', video_id, 'Getting bearer token', headers={
  File "/home/capture-au/.local/lib/python3.9/site-packages/yt_dlp/extractor/common.py", line 976, in _download_json
    res = self._download_json_handle(
  File "/home/capture-au/.local/lib/python3.9/site-packages/yt_dlp/extractor/common.py", line 962, in _download_json_handle
    return self._parse_json(
  File "/home/capture-au/.local/lib/python3.9/site-packages/yt_dlp/extractor/common.py", line 991, in _parse_json
    raise ExtractorError(errmsg, cause=ve)
yt_dlp.utils.ExtractorError: tpv210909jrytv: Failed to parse JSON  (caused by JSONDecodeError('Expecting value: line 1 column 1 (char 0)')); please report this issue on  https://github.com/yt-dlp/yt-dlp . Make sure you are using the latest version; see  https://github.com/yt-dlp/yt-dlp  on how to update. Be sure to call yt-dlp with the --verbose flag and include its complete output.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/capture-au/.local/lib/python3.9/site-packages/yt_dlp/YoutubeDL.py", line 1318, in wrapper
    return func(self, *args, **kwargs)
  File "/home/capture-au/.local/lib/python3.9/site-packages/yt_dlp/YoutubeDL.py", line 1343, in __extract_info
    ie_result = ie.extract(url)
  File "/home/capture-au/.local/lib/python3.9/site-packages/yt_dlp/extractor/common.py", line 622, in extract
    raise type(e)(e.msg, **kwargs)
yt_dlp.utils.ExtractorError: [TenPlay] tpv210909jrytv: tpv210909jrytv: Failed to parse JSON  (caused by JSONDecodeError('Expecting value: line 1 column 1 (char 0)')); please report this issue on  https://github.com/yt-dlp/yt-dlp . Make sure you are using the latest version; see  https://github.com/yt-dlp/yt-dlp  on how to update. Be sure to call yt-dlp with the --verbose flag and include its complete output. (caused by JSONDecodeError('Expecting value: line 1 column 1 (char 0)')); please report this issue on  https://github.com/yt-dlp/yt-dlp . Make sure you are using the latest version; see  https://github.com/yt-dlp/yt-dlp  on how to update. Be sure to call yt-dlp with the --verbose flag and include its complete output.

[debug] [TenPlay] Extracting URL: https://10play.com.au/mirror-mirror/episodes/season-1/episode-1/tpv211105pbzld
[TenPlay] tpv211105pbzld: Getting bearer token
ERROR: [TenPlay] tpv211105pbzld: tpv211105pbzld: Failed to parse JSON  (caused by JSONDecodeError('Expecting value: line 1 column 1 (char 0)')); please report this issue on  https://github.com/yt-dlp/yt-dlp . Make sure you are using the latest version; see  https://github.com/yt-dlp/yt-dlp  on how to update. Be sure to call yt-dlp with the --verbose flag and include its complete output. (caused by JSONDecodeError('Expecting value: line 1 column 1 (char 0)')); please report this issue on  https://github.com/yt-dlp/yt-dlp . Make sure you are using the latest version; see  https://github.com/yt-dlp/yt-dlp  on how to update. Be sure to call yt-dlp with the --verbose flag and include its complete output.
Traceback (most recent call last):
  File "/home/capture-au/.local/lib/python3.9/site-packages/yt_dlp/extractor/common.py", line 987, in _parse_json
    return json.loads(json_string)
  File "/home/linuxbrew/.linuxbrew/opt/python@3.9/lib/python3.9/json/__init__.py", line 346, in loads
    return _default_decoder.decode(s)
  File "/home/linuxbrew/.linuxbrew/opt/python@3.9/lib/python3.9/json/decoder.py", line 337, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/home/linuxbrew/.linuxbrew/opt/python@3.9/lib/python3.9/json/decoder.py", line 355, in raw_decode
    raise JSONDecodeError("Expecting value", s, err.value) from None
json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/capture-au/.local/lib/python3.9/site-packages/yt_dlp/extractor/common.py", line 596, in extract
    ie_result = self._real_extract(url)
  File "/home/capture-au/.local/lib/python3.9/site-packages/yt_dlp/extractor/tenplay.py", line 65, in _real_extract
    _token = self._get_bearer_token(content_id)
  File "/home/capture-au/.local/lib/python3.9/site-packages/yt_dlp/extractor/tenplay.py", line 55, in _get_bearer_token
    data = self._download_json('https://10play.com.au/api/user/auth', video_id, 'Getting bearer token', headers={
  File "/home/capture-au/.local/lib/python3.9/site-packages/yt_dlp/extractor/common.py", line 976, in _download_json
    res = self._download_json_handle(
  File "/home/capture-au/.local/lib/python3.9/site-packages/yt_dlp/extractor/common.py", line 962, in _download_json_handle
    return self._parse_json(
  File "/home/capture-au/.local/lib/python3.9/site-packages/yt_dlp/extractor/common.py", line 991, in _parse_json
    raise ExtractorError(errmsg, cause=ve)
yt_dlp.utils.ExtractorError: tpv211105pbzld: Failed to parse JSON  (caused by JSONDecodeError('Expecting value: line 1 column 1 (char 0)')); please report this issue on  https://github.com/yt-dlp/yt-dlp . Make sure you are using the latest version; see  https://github.com/yt-dlp/yt-dlp  on how to update. Be sure to call yt-dlp with the --verbose flag and include its complete output.
Traceback (most recent call last):
  File "/home/capture-au/.local/lib/python3.9/site-packages/yt_dlp/extractor/common.py", line 987, in _parse_json
    return json.loads(json_string)
  File "/home/linuxbrew/.linuxbrew/opt/python@3.9/lib/python3.9/json/__init__.py", line 346, in loads
    return _default_decoder.decode(s)
  File "/home/linuxbrew/.linuxbrew/opt/python@3.9/lib/python3.9/json/decoder.py", line 337, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/home/linuxbrew/.linuxbrew/opt/python@3.9/lib/python3.9/json/decoder.py", line 355, in raw_decode
    raise JSONDecodeError("Expecting value", s, err.value) from None
json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/capture-au/.local/lib/python3.9/site-packages/yt_dlp/extractor/common.py", line 596, in extract
    ie_result = self._real_extract(url)
  File "/home/capture-au/.local/lib/python3.9/site-packages/yt_dlp/extractor/tenplay.py", line 65, in _real_extract
    _token = self._get_bearer_token(content_id)
  File "/home/capture-au/.local/lib/python3.9/site-packages/yt_dlp/extractor/tenplay.py", line 55, in _get_bearer_token
    data = self._download_json('https://10play.com.au/api/user/auth', video_id, 'Getting bearer token', headers={
  File "/home/capture-au/.local/lib/python3.9/site-packages/yt_dlp/extractor/common.py", line 976, in _download_json
    res = self._download_json_handle(
  File "/home/capture-au/.local/lib/python3.9/site-packages/yt_dlp/extractor/common.py", line 962, in _download_json_handle
    return self._parse_json(
  File "/home/capture-au/.local/lib/python3.9/site-packages/yt_dlp/extractor/common.py", line 991, in _parse_json
    raise ExtractorError(errmsg, cause=ve)
yt_dlp.utils.ExtractorError: tpv211105pbzld: Failed to parse JSON  (caused by JSONDecodeError('Expecting value: line 1 column 1 (char 0)')); please report this issue on  https://github.com/yt-dlp/yt-dlp . Make sure you are using the latest version; see  https://github.com/yt-dlp/yt-dlp  on how to update. Be sure to call yt-dlp with the --verbose flag and include its complete output.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/capture-au/.local/lib/python3.9/site-packages/yt_dlp/YoutubeDL.py", line 1318, in wrapper
    return func(self, *args, **kwargs)
  File "/home/capture-au/.local/lib/python3.9/site-packages/yt_dlp/YoutubeDL.py", line 1343, in __extract_info
    ie_result = ie.extract(url)
  File "/home/capture-au/.local/lib/python3.9/site-packages/yt_dlp/extractor/common.py", line 622, in extract
    raise type(e)(e.msg, **kwargs)
yt_dlp.utils.ExtractorError: [TenPlay] tpv211105pbzld: tpv211105pbzld: Failed to parse JSON  (caused by JSONDecodeError('Expecting value: line 1 column 1 (char 0)')); please report this issue on  https://github.com/yt-dlp/yt-dlp . Make sure you are using the latest version; see  https://github.com/yt-dlp/yt-dlp  on how to update. Be sure to call yt-dlp with the --verbose flag and include its complete output. (caused by JSONDecodeError('Expecting value: line 1 column 1 (char 0)')); please report this issue on  https://github.com/yt-dlp/yt-dlp . Make sure you are using the latest version; see  https://github.com/yt-dlp/yt-dlp  on how to update. Be sure to call yt-dlp with the --verbose flag and include its complete output.
```

**Comment by pukkandan** [comment]
@king-millez

**Comment by 2011** [comment]
I get slightly different error messages, but this looks like the same issue:

```
$ /tmp/yt-dlp-2022-07-18 -F --cookies /home/user/cookies.text --list-subs -n -v 'https://10play.com.au/hunted/episodes/season-1/hunted-s1-ep-1/tpv220713igmqn'
[debug] Command-line config: ['-F', '--cookies', '/home/user/cookies.text', '--list-subs', '-n', '-v', 'https://10play.com.au/hunted/episodes/season-1/hunted-s1-ep-1/tpv220713igmqn']
[debug] Encodings: locale UTF-8, fs utf-8, pref UTF-8, out utf-8, error utf-8, screen utf-8
[debug] yt-dlp version 2022.07.18 [135f05e] (zip)
[debug] Python 3.9.13 (CPython 64bit) - Linux-5.15.41-with-glibc2.34 (glibc 2.34)
[debug] Checking exe version: ffmpeg -bsfs
[debug] Checking exe version: ffprobe -bsfs
[debug] exe versions: ffmpeg 4.4.2 (setts), ffprobe 4.4.2
[debug] Optional libraries: Crypto-3.14.1, certifi-3021.03.16
[debug] Proxy map: {}
[debug] [TenPlay] Extracting URL: https://10play.com.au/hunted/episodes/season-1/hunted-s1-ep-1/tpv220713igmqn
[TenPlay] tpv220713igmqn: Downloading JSON metadata
[TenPlay] tpv220713igmqn: Getting bearer token
ERROR: [TenPlay] tpv220713igmqn: Unable to download JSON metadata: Remote end closed connection without response (caused by RemoteDisconnected('Remote end closed connection without response')); please report this issue on  https://github.com/yt-dlp/yt-dlp/issues?q= , filling out the appropriate issue template. Confirm you are on the latest version using  yt-dlp -U
  File "/tmp/yt-dlp-2022-07-18/yt_dlp/extractor/common.py", line 644, in extract
    ie_result = self._real_extract(url)
  File "/tmp/yt-dlp-2022-07-18/yt_dlp/extractor/tenplay.py", line 90, in _real_extract
    _token = self._get_bearer_token(content_id)
  File "/tmp/yt-dlp-2022-07-18/yt_dlp/extractor/tenplay.py", line 75, in _get_bearer_token
    data = self._download_json('https://10play.com.au/api/user/auth', video_id, 'Getting bearer token', headers={
  File "/tmp/yt-dlp-2022-07-18/yt_dlp/extractor/common.py", line 1004, in download_content
    res = getattr(self, download_handle.__name__)(url_or_request, video_id, **kwargs)
  File "/tmp/yt-dlp-2022-07-18/yt_dlp/extractor/common.py", line 968, in download_handle
    res = self._download_webpage_handle(
  File "/tmp/yt-dlp-2022-07-18/yt_dlp/extractor/common.py", line 838, in _download_webpage_handle
    urlh = self._request_webpage(url_or_request, video_id, note, errnote, fatal, data=data, headers=headers, query=query, expected_status=expected_status)
  File "/tmp/yt-dlp-2022-07-18/yt_dlp/extractor/common.py", line 795, in _request_webpage
    raise ExtractorError(errmsg, cause=err)

  File "/tmp/yt-dlp-2022-07-18/yt_dlp/extractor/common.py", line 777, in _request_webpage
    return self._downloader.urlopen(self._create_request(url_or_request, data, headers, query))
  File "/tmp/yt-dlp-2022-07-18/yt_dlp/YoutubeDL.py", line 3639, in urlopen
    return self._opener.open(req, timeout=self._socket_timeout)
  File "/usr/lib/python3.9/urllib/request.py", line 517, in open
    response = self._open(req, data)
  File "/usr/lib/python3.9/urllib/request.py", line 534, in _open
    result = self._call_chain(self.handle_open, protocol, protocol +
  File "/usr/lib/python3.9/urllib/request.py", line 494, in _call_chain
    result = func(*args)
  File "/tmp/yt-dlp-2022-07-18/yt_dlp/utils.py", line 1455, in https_open
    return self.do_open(
  File "/usr/lib/python3.9/urllib/request.py", line 1350, in do_open
    r = h.getresponse()
  File "/usr/lib/python3.9/http/client.py", line 1377, in getresponse
    response.begin()
  File "/usr/lib/python3.9/http/client.py", line 320, in begin
    version, status, reason = self._read_status()
  File "/usr/lib/python3.9/http/client.py", line 289, in _read_status
    raise RemoteDisconnected("Remote end closed connection without"
http.client.RemoteDisconnected: Remote end closed connection without response
```

A couple of side questions if you get a chance (I have never used .netrc or cookies with yt-dlp before).

First, why does the extractor have the name "Tenplay" and the .netrc file want "10play" (I give the program credit for guiding me to the correct choice, though)? :)

Second, do I need to use the cookies option (in this case, or when in general)? I don't particularly care about saving them (or should I - will saved cookies eliminate having to supply credentials for future downloads?).

**Comment by nyuszika7h** [comment]
> why does the extractor have the name "Tenplay"
Python module/class names can’t start with a number.

**Comment by futurenow888** [comment]
Thanks for posting this error. Using version version 2022.11.11, it's still a 400 Bad Request, but the output is slightly different:

```
yt-dlp.exe -F -v -n https://10play.com.au/mirror-mirror/episodes/season-1/episode-2/tpv210909jrytv
[debug] Command-line config: ['-F', '-v', '-n', 'https://10play.com.au/mirror-mirror/episodes/season-1/episode-2/tpv210909jrytv']
[debug] Encodings: locale cp1252, fs utf-8, pref cp1252, out utf-8, error utf-8, screen utf-8
[debug] yt-dlp version 2022.11.11 [8b64402]
[debug] Python 3.8.10 (CPython AMD64 64bit)
[debug] exe versions: none
[debug] Optional libraries: Cryptodome-3.15.0, brotli-1.0.9, certifi-2022.09.24, mutagen-1.46.0, sqlite3-2.6.0, websockets-10.4
[debug] Proxy map: {}
[debug] Loaded 1723 extractors
[debug] [TenPlay] Extracting URL: https://10play.com.au/mirror-mirror/episodes/season-1/episode-2/tpv210909jrytv
[TenPlay] tpv210909jrytv: Downloading JSON metadata
[TenPlay] tpv210909jrytv: Getting bearer token
ERROR: [TenPlay] tpv210909jrytv: Unable to download JSON metadata: HTTP Error 400: Bad Request (caused by <HTTPError 400: 'Bad Request'>); please report this issue on  https://github.com/yt-dlp/yt-dlp/issues?q= , filling out the appropriate issue template. Confirm you are on the latest version using  yt-dlp -U
  File "yt_dlp\extractor\common.py", line 674, in extract
  File "yt_dlp\extractor\tenplay.py", line 90, in _real_extract
  File "yt_dlp\extractor\tenplay.py", line 75, in _get_bearer_token
  File "yt_dlp\extractor\common.py", line 1034, in download_content
  File "yt_dlp\extractor\common.py", line 998, in download_handle
  File "yt_dlp\extractor\common.py", line 868, in _download_webpage_handle
  File "yt_dlp\extractor\common.py", line 825, in _request_webpage

  File "yt_dlp\extractor\common.py", line 807, in _request_webpage
  File "yt_dlp\YoutubeDL.py", line 3692, in urlopen
  File "urllib\request.py", line 531, in open
  File "urllib\request.py", line 640, in http_response
  File "urllib\request.py", line 569, in error
  File "urllib\request.py", line 502, in _call_chain
  File "urllib\request.py", line 649, in http_error_default
urllib.error.HTTPError: HTTP Error 400: Bad Request

```

**Comment by futurenow888** [comment]
The same error occurs with version 2023.01.06, with slightly different python script line numbers:

```
yt-dlp.exe -F -v -n https://10play.com.au/mirror-mirror/episodes/season-1/episode-2/tpv210909jrytv
[debug] Command-line config: ['-F', '-v', '-n', 'https://10play.com.au/mirror-mirror/episodes/season-1/episode-2/tpv210909jrytv']
[debug] Encodings: locale cp1252, fs utf-8, pref cp1252, out utf-8, error utf-8, screen utf-8
[debug] yt-dlp version 2023.01.06 [6becd25] (win_exe)
[debug] Python 3.8.10 (CPython AMD64 64bit)
[debug] exe versions: none
[debug] Optional libraries: Cryptodome-3.16.0, brotli-1.0.9, certifi-2022.12.07, mutagen-1.46.0, sqlite3-2.6.0, websockets-10.4
[debug] Proxy map: {}
[debug] Loaded 1760 extractors
[TenPlay] Extracting URL: https://10play.com.au/mirror-mirror/episodes/season-1/episode-2/tpv210909jrytv
[TenPlay] tpv200401iawhv: Downloading JSON metadata
[TenPlay] tpv200401iawhv: Getting bearer token
ERROR: [TenPlay] tpv200401iawhv: Unable to download JSON metadata: HTTP Error 400: Bad Request (caused by <HTTPError 400: 'Bad Request'>); please report this issue on  https://github.com/yt-dlp/yt-dlp/issues?q= , filling out the appropriate issue template. Confirm you are on the latest version using  yt-dlp -U
  File "yt_dlp\extractor\common.py", line 680, in extract
  File "yt_dlp\extractor\tenplay.py", line 90, in _real_extract
  File "yt_dlp\extractor\tenplay.py", line 75, in _get_bearer_token
  File "yt_dlp\extractor\common.py", line 1034, in download_content
  File "yt_dlp\extractor\common.py", line 998, in download_handle
  File "yt_dlp\extractor\common.py", line 868, in _download_webpage_handle
  File "yt_dlp\extractor\common.py", line 825, in _request_webpage

  File "yt_dlp\extractor\common.py", line 807, in _request_webpage
  File "yt_dlp\YoutubeDL.py", line 3730, in urlopen
  File "urllib\request.py", line 531, in open
  File "urllib\request.py", line 640, in http_response
  File "urllib\request.py", line 569, in error
  File "urllib\request.py", line 502, in _call_chain
  File "urllib\request.py", line 649, in http_error_default
urllib.error.HTTPError: HTTP Error 400: Bad Request
```

**Comment by chrisdma** [comment]
Any update on this issue for 10play?

**Comment by futurenow888** [comment]
The same error occurs with version stable@2023.03.04, with slightly different python script line numbers:

```
yt-dlp.exe -F -v -n https://10play.com.au/mirror-mirror/episodes/season-1/episode-2/tpv210909jrytv
[debug] Command-line config: ['-F', '-v', '-n', 'https://10play.com.au/mirror-mirror/episodes/season-1/episode-2/tpv210909jrytv']
[debug] Encodings: locale cp1252, fs utf-8, pref cp1252, out utf-8, error utf-8, screen utf-8
[debug] yt-dlp version stable@2023.03.04 [392389b7d]
[debug] Python 3.8.10 (CPython AMD64 64bit)
[debug] exe versions: none
[debug] Optional libraries: Cryptodome-3.17, brotli-1.0.9, certifi-2022.12.07, mutagen-1.46.0, sqlite3-2.6.0, websockets-10.4
[debug] Proxy map: {}
[debug] Loaded 1786 extractors
[TenPlay] Extracting URL: https://10play.com.au/mirror-mirror/episodes/season-1/episode-2/tpv210909jrytv
[TenPlay] tpv210909jrytv: Downloading JSON metadata
[TenPlay] tpv210909jrytv: Getting bearer token
ERROR: [TenPlay] tpv210909jrytv: Unable to download JSON metadata: HTTP Error 400: Bad Request (caused by <HTTPError 400: 'Bad Request'>); please report this issue on  https://github.com/yt-dlp/yt-dlp/issues?q= , filling out the appropriate issue template. Confirm you are on the latest version using  yt-dlp -U
  File "yt_dlp\extractor\common.py", line 694, in extract
  File "yt_dlp\extractor\tenplay.py", line 90, in _real_extract
  File "yt_dlp\extractor\tenplay.py", line 75, in _get_bearer_token
  File "yt_dlp\extractor\common.py", line 1048, in download_content
  File "yt_dlp\extractor\common.py", line 1012, in download_handle
  File "yt_dlp\extractor\common.py", line 882, in _download_webpage_handle
  File "yt_dlp\extractor\common.py", line 839, in _request_webpage

  File "yt_dlp\extractor\common.py", line 821, in _request_webpage
  File "yt_dlp\YoutubeDL.py", line 3742, in urlopen
  File "urllib\request.py", line 531, in open
  File "urllib\request.py", line 640, in http_response
  File "urllib\request.py", line 569, in error
  File "urllib\request.py", line 502, in _call_chain
  File "urllib\request.py", line 649, in http_error_default
urllib.error.HTTPError: HTTP Error 400: Bad Request

**Comment by futurenow888** [comment]
Could this issue be closed? This appears to be the same as [#8110](https://github.com/yt-dlp/yt-dlp/issues/8110) , with the same fix.
